SimPEG/Mesh/CylMesh.py

```diff
@@ -367,6 +367,52 @@ def aveF2CCV(self):
                                           'yet implemented')
         return self._aveF2CCV
 
+
+    def getInterpolationMat(self, loc, locType='CC', zerosOutside=False):
```

Comment:
I guess you answered it in your notes above. CCv needs specific treatment in the Cyl mesh.